lib/ProductOpener/Ingredients.pm

```diff
@@ -4930,13 +4930,11 @@ my %phrases_after_ingredients_list = (
 	],
 
 	hr => [
-		'bez konzervans',    # without preservatives
```

Comment:
Just curious: why all the removals? they were not relevant / useful?

Reply:
I added them long time ago.

'bez konzervans' and 'izvor dijetalnih vlakana' are labels. If I understand well, they can be extracted and put to labels tags

čuvati is problematic. It seems not to be working all the time. Hence, I will try to add both upper and lower cases for now. Is there a way to test it locally? (when I upload picture locally, I cannot extract text from it). Additionally, čuvati literally means "to store", which will not appears in the middle of ingredients, so, this is why I removed optional text that appears after: (zatvoreno na|pri sobnoj temperaturi|u zamrzivaču na).

Finally, the first line, it also contain čuvati. Optional text before is not clear, it seems to me that it could be added as new line. I will remove it for now, and wait to see again these occurences in the future and eventually add them again.